src/NServiceBus.Callbacks/ExtendableOptionsExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace NServiceBus
+{
+    using System.Threading;
+    using NServiceBus.Extensibility;
+
+    static class ExtendableOptionsExtensions
+    {
+        public static void RegisterTokenInternal(this ExtendableOptions options, CancellationToken cancellationToken)
+        {
+            var extensions = options.GetExtensions();
+            UpdateRequestResponseCorrelationTableBehavior.RequestResponseParameters data;
```

Comment:
>Should I move SendLocalOptionsExtensions and SendOptionsExtensions including ExtendableOptionsExtensions to TransientRequestResponseExtensions?

Lets leave as is since I hope to kill the SendLocalOptions as soon this one is in?